recipes/bind/bind.inc

```diff
@@ -54,6 +55,6 @@ inherit binconfig
 BINCONFIG_FILES = "${bindir}/isc-config.sh"
 FILES_${PN}-dev += "${bindir}/isc-config.sh"
 DEPENDS_${PN}-dev = "libssl libcrypto ${DEPENDS_LIBC}"
-
-PACKAGES =+ "${PN}-utils "
+FILES_${PN}-config = "${sysconfdir}/bind/named.conf.local"
```

Comment:
that's fine, i just need to check that we are not still pushing some oe-lite default configuration package as part of the non-config packages.